oak/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+cc_library(
+    name = "oak",
+    hdrs = glob(["include/**/*.hpp"]),
+    srcs = glob(["src/**"], exclude=["src/**/*_test.cpp"]),
+    includes = [ "include" ],
+    copts = [
+        "-Ioak/src"
```

Comment:
why is this necessary?

Reply:
It's probably not...

Reply:
How would you define *local includes*. There are headers not part of the public interface that I want to include and they are in the `src` folder.

Reply:
https://github.com/bazelbuild/bazel/issues/15553#issuecomment-1134954324

So apparently this is the only way right now. A bit disappointing.

Reply:
just add the private headers to the srcs glob as well (we are doing this as well in various places).

Reply:
Ok, it works if I include relative to the source file.